pyproject.toml

```diff
@@ -66,7 +66,7 @@ greenlet = "^1.1.2"
 toml = "^0.10.2"
 responses = "^0.21.0"
 pytest-mock = "^3.8.2"
-flake8 = ">=4,<5"
+flake8 = ">=5,<6"
```

Comment:
If I'm not mistaken, with `"rangeStrategy": "widen"`, this should become `>=4,<6`.